src/content/release-notes.mdx

```diff
@@ -5,6 +5,29 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.16.0
+
+20 December 2023
+
+### Improvements
+
+- Okteto garbage collector cronjob now resolves buildkit and ingress hosts to their internal IPs  <!-- 7432 -->
+- Registry Credentials cronjob only update dockerconfig secret if the credentials change.  <!-- 7474 -->
+- Okteto Daemon now update kubelet docker config.json file in nodes reactively when the static docker config secret changes. Bumps okteto/daemon to 0.5.1
+- Fix network policies configuration to limit buildkit access to the nginx controller  <!-- 7505 -->
+- Add Okteto Insights Token to the Admin UI  <!-- 7473 -->
+- Remove customer data from mixpanel events  <!-- 7390 -->
```

Comment:
Can we skip this one? We aren't sending mixpannel events in SH, I don't want to give the impression that we had a security leak here